src/superlifter/core.cljc

```diff
@@ -178,9 +178,18 @@
                      %))
   context)
 
+(defn- stop-bucket! [bucket]
```

Comment:
We could also call this from `stop!`
https://github.com/oliyh/superlifter/blob/ded9eef3b0db9af07962583934a7e3f4c126777a/src/superlifter/core.cljc#L239